README.md

```diff
@@ -39,20 +39,34 @@ It will serve the following Composer repository at `/packages.json` automagicall
 
 ## Installation
 
-Create the project:
+### 1. Create the project:
 
 ```
 composer create-project rarst/release-belt
 ```
 
-Place release ZIPs into `/releases/[type]/[vendor]/`.
+### 2. Place release ZIPs into `/releases/[type]/[vendor]/`. 
+`[type]` could be e.g. "symfony-bundle", "wordpress-plugin or "typo3-cms-extension".
```

Comment:
Technically the explicitly supported types right now are `library`, `wordpress-plugin`, and `wordpress-theme`, could you please use these for examples?

Arbitrary type would work, but will not do any additional tweaking (such as requiring `composer/installers`).